roles/edpm_ovn/molecule/default/molecule.yml

```diff
@@ -26,6 +26,7 @@ scenario:
   - prepare
   - converge
   - verify
```

Comment:
Maybe it would be also good to ensure that in the default scenario both chassis_as_gateway and chassis_as_external external_ids aren't set at all?

Reply:
Sure, I can do that

Reply:
Added